node/packages/sdk-schema/.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const path = require('path');
+
+const projectDir = path.resolve(__dirname, '../..');
+
+module.exports = {
+  extends: path.resolve(projectDir, '.eslintrc.js'),
+  rules: {
+    'import/no-extraneous-dependencies': [
+      'error',
+      {
+        devDependencies: ['**/scripts/**', '**/test/**'],
+        packageDir: [projectDir, path.resolve(projectDir, 'packages/sdk-schema')],
+      },
+    ],
+  },
+  ignorePatterns: ['rollup.config.js', '!*.test.js', '**/test/**/*.test.js'],
```

Comment:
We talked about this the other day. If we include the tests in the lint config, then every single run of eslint will need to have the protobufs built and the files generated. Doesn't seem like a good idea to do that.

Reply:
I understand that issue is that lint complains on missing modules (?)

If that's the case, I think it's better to add lint ignore comments above such require, and keep the linting

Linting is really really important, as it can pick some important issues which can be missed during manual review.